File: All Codes/URI_1036.java
import java.util.Scanner;
/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */

public class URI_1036 {
    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            double t;
            double a = input.nextDouble();
            double b = input.nextDouble();
            double c = input.nextDouble();

            if(((b * b) - 4 * a * c) < 0 || a == 0){
                System.out.printf("Impossivel calcular\n");
            }
            else{
                t = Math.sqrt((b * b) - 4 * a * c);
                System.out.printf("R1 = %.5f\nR2 = %.5f\n", ((-b + t) / (2 * a)), ((-b - t) / (2 * a)));
            }
        }
    }
}

File: All Codes/URI_1036.py
a = [float(x) for x in input().split()]
# a[0] = a
# a[1] = b
# a[2] = c

# cal = (b * b) - 4 * a * c 
cal = (a[1] * a[1]) - 4 * a[0] * a[2]

# (b * b) - 4 * a * c < 0 || a == 0
con = False if cal < 0 or a[0] == 0 else True

if con:
    root_sqr = cal ** 0.5
    pos = (-a[1] + root_sqr) / (2 * a[0])
    neg = (-a[1] - root_sqr) / (2 * a[0])
    print(f"R1 = {pos:.5f}\nR2 = {neg:.5f}")
else:
    print("Impossivel calcular")
